docker/webserver/config/ckan_config.sh

```diff
@@ -14,6 +14,10 @@
 
 set -eu
 
+# activate the virtual environment
+
+. /usr/lib/ckan/bin/activate
```

Comment:
Since this is done by the entrypoint, not sure if its necessary.

Reply:
Removed.